src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+import { HStack, Flex } from "@chakra-ui/react"
+
+export interface FooterProps {
+  children: JSX.Element
```

Comment:
could we use `PropsWithChildren` from react here for btr typing?

Reply:
Thanks for the suggestion, changed in https://github.com/isomerpages/isomercms-frontend/pull/932/commits/7faa682fd7af5140926947afef3520d12daa67fb